docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       - SPRING_DATASOURCE_URL=jdbc:postgresql://localhost:5432/local_nakadi_db
       - SPRING_DATASOURCE_USERNAME=nakadi
       - SPRING_DATASOURCE_PASSWORD=nakadi
+      - NAKADI_SUBSCRIPTION_MAX_PARTITIONS=30
```

Comment:
@antban I understand that, but in order not to pollute the docker-compose.yml, you can define this property for test spring profile.